resources/css/layouts/_app.css

```diff
@@ -1 +1,9 @@
 /* This file contains all styles related to the application layout. */
+
+ul {
+    margin-block: 0;
+}
```

Comment:
I'd rather have this in `base/_base.css`. I think that's the right place for it rather than the app layout.